source/brailleTables.py

```diff
@@ -452,6 +452,9 @@ def listTables():
 addTable("pu-in-g1.utb", _("Punjabi grade 1"))
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
+addTable("ro-g0.utb", _("Romanian 6 dot"))
+# Translators: The name of a braille table displayed in the
+# braille settings dialog.
 addTable("ro.ctb", _("Romanian"))

```

Comment:
Should this one be renamed in the GUI?

Reply:
I'm not sure.

Reply:
Having a 6 dot and a nothing is a bit confusing.

Reply:
Hmmm, how about Romanian (legacy)?

Reply:
Perhaps it's worth asking our romanian translators: https://github.com/nvaccess/mrconfig/blob/master/scripts/addressData.py#L358

Reply:
Have we heard anything back yet?

Reply:
I'm assuming we haven't heard anything back yet. In absence of a response, lets merge and fix the strings later if required?

Reply:
No response yet, sounds reasonable to me